### PR TITLE
Added heuristics docs

### LINKED ISSUE #69: Document coding styles and IDE integrations
We have some Go coding style rules that we decided on in the Wharf team a while ago, but this was before we went open source. 

These notes needs to be moved to iver-wharf.github.io, cleaned up, and then possibly linked in each repo's CONTRIBUTING.md

Ask me, @jilleJr, where to find these private notes if you wish to take on this task.

### PR DESCRIPTION
**Pull request by jilleJr**
Our agreed upon heuristics that we try to follow when developing Wharf.

Finally got around to writing this down. This was the last note from our closed-sourced notebook, and we can finally forget about it and only refer to our open-sourced notebooks from now on.

Once this is merged, I will open another PR for a CONTRIBUTING.md file in wharf-api to begin with.

Closes #69

### CODE REVIEW

**Review by Alexamakans — APPROVED** [review]

File: docs/index.html [hunk 1]
```diff
@@ -47,6 +47,7 @@
   <script src="//cdn.jsdelivr.net/npm/prismjs@1/components/prism-diff.min.js"></script>
   <script src="//cdn.jsdelivr.net/npm/prismjs@1/components/prism-docker.min.js"></script>
   <script src="//cdn.jsdelivr.net/npm/prismjs@1/components/prism-go.min.js"></script>
+  <script src="//cdn.jsdelivr.net/npm/prismjs@1/components/prism-http.min.js"></script>
```
Comment (fredx30) [inline docs/index.html:50]: Seems unrelated- verify?

**Review by fredx30 — APPROVED** [review]

File: docs/index.html (hunk 1 above)
Comment (Alexamakans) [reply docs/index.html:50]: It's from the html code block at `docs/development/heuristics/rest-api.md#L33` I would assume.

Comment (jilleJr) [reply docs/index.html:50]: Yea it's syntax highlighting for `http` code blocks

https://github.com/iver-wharf/iver-wharf.github.io/blob/88c3c3ac108e076bb41ebe357f6938136eb10e8f/docs/development/heuristics/rest-api.md#L33-L41
